src/lava/magma/core/run_configs.py

```diff
@@ -70,19 +70,21 @@ def select(self,
 class Loihi1SimCfg(RunConfig):
     """Run configuration selects appropriate ProcessModel -- either
     `SubProcessModel` for a hierarchical Process or else a `PyProcessModel`
-    for a standard Process.
+    for a standard Process. Implements and overrides `select()` method of the
```

Comment:
'Implements AND overrides' is redundant. The parent class is abstract and only provides a virtual select method (we could make it an @abstractmethod). Thus we only implement it.

Reply:
It is actually entirely unnecessary to say this because it is a MUST that a concrete child class implements the abstract method of its parent.

Reply:
Removed.